NightscoutUploadKit/NightscoutUploader.swift

```diff
@@ -483,6 +565,32 @@ public class NightscoutUploader {
         }
     }
     
```

Comment:
This is not used yet but this can help us when ready to decode JSON from these new concurrent APIs.